notebooks/500__jupyterhub/elyra/run-pipelines-on-data-science-pipelines/hello-generic-world.pipeline

```diff
@@ -253,6 +277,7 @@
               "content": "Download the data",
               "associated_id_refs": [
                 {
+                  "id": "cb879ebf-6a33-4504-beb0-d78dd507e7d9",
```

Comment:
what are these IDs?

Reply:
these are associated task ids, which are generated by elyra.
removing them as they would be re-created anyway on execution in the cluster.